dev/_sass/_bootstrap/_utilities.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../node_modules/bootstrap/scss/utilities/align";
+@import "../../node_modules/bootstrap/scss/utilities/background";
+@import "../../node_modules/bootstrap/scss/utilities/borders";
+@import "../../node_modules/bootstrap/scss/utilities/clearfix";
+@import "../../node_modules/bootstrap/scss/utilities/display";
+@import "../../node_modules/bootstrap/scss/utilities/embed";
+@import "../../node_modules/bootstrap/scss/utilities/flex";
+@import "../../node_modules/bootstrap/scss/utilities/float";
+@import "../../node_modules/bootstrap/scss/utilities/position";
+@import "../../node_modules/bootstrap/scss/utilities/screenreaders";
+@import "../../node_modules/bootstrap/scss/utilities/shadows";
+@import "../../node_modules/bootstrap/scss/utilities/sizing";
+@import "../../node_modules/bootstrap/scss/utilities/spacing";
+@import "text";
```

Comment:
Why text, not `../../node_modules/bootstrap/scss/utilities/text`

Reply:
In text.scss file I removed `@include text-hide();` cause it made an error

Reply:
Oh got it!